packages/core/src/RenderPipeline/MaskManager.ts

```diff
@@ -34,12 +35,12 @@ export class MaskManager {
   }
 
   buildMaskRenderElement(
-    element: SubRenderElement,
+    elementOrNumber: SubRenderElement | number,
```

Comment:
The numerical parameter here can actually only be 0. It would be better to add a reset flag or change the name to a more suitable one.

Reply:
done